workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/Run.java

```diff
@@ -193,9 +195,6 @@ public void shutdown() {
 		 * Send exit messages to the other driver nodes
 		 */
 		for (String hostname : hosts) {
-			if (hostname == statsHost) {
-				continue;
-			}
```

Comment:
Is it ok to completely remove this check?  Could it check for "localhost" on runStatsHost?

Reply:
I added back the check, but comparing against workloadStatsHost.  This check doesn't matter for Kubernetes, but for Docker it will matter if the workload controller also runs a driver node (which is the way it currently works).